src/Mod/TechDraw/App/DrawDimHelper.cpp

```diff
@@ -59,14 +59,15 @@
 
 using namespace TechDraw;
 
-void DrawDimHelper::makeExtentDim(DrawViewPart* dvp, std::vector<std::string> edgeNames,
+DrawViewDimExtent*
+DrawDimHelper::makeExtentDim(DrawViewPart* dvp, std::vector<std::string> edgeNames,
                                   int direction)
 {
     //    Base::Console().Message("DDH::makeExtentDim() - dvp: %s edgeNames: %d\n",
     //                            dvp->Label.getValue(), edgeNames.size());
-    if (!dvp) {
-        return;
-    }
+    //if (!dvp) {
+    //    return;
+    //}
```

Comment:
I would leave the check for a valid dvp in place.  If you make a reference with a null object it is sure to cause trouble later.  Just return nullptr here and check for that at time of use.